pkg/process/checks/process.go

```diff
@@ -40,15 +40,25 @@ const (
 // NewProcessCheck returns an instance of the ProcessCheck.
 func NewProcessCheck(config ddconfig.ConfigReader) *ProcessCheck {
 	var extractors []metadata.Extractor
-	if workloadMetaExtractor.Enabled(config) {
-		extractors = append(extractors, workloadMetaExtractor.NewWorkloadMetaExtractor())
+	var wlmServer *workloadmeta.GRPCServer
+	if workloadmeta.Enabled(config) {
+		wlmExtractor := workloadmeta.NewWorkloadMetaExtractor(config)
+		srv := workloadmeta.NewGRPCServer(config, wlmExtractor)
+		err := srv.Start()
+		if err != nil {
+			_ = log.Error("Failed to start the workload meta gRPC server:", err)
```

Comment:
[minor] let's just ignore the error completely from the logger as this is the idiomatic way of using a logger
```suggestion
			log.Error("Failed to start the workload meta gRPC server:", err)
```

Reply:
Fixed on https://github.com/DataDog/datadog-agent/pull/17594/commits/c7b47f9e299d2e5faed1e7de18c76499f36bfa78.